From 6c29279caeb5dc99e9bdc6dd6fcba6772990bdcb Mon Sep 17 00:00:00 2001
From: Sergei Reznikov <diver@gelios.net>
Date: Fri, 15 Dec 2017 12:35:46 +0300
Subject: Disable kauth_ktexteditor_helper for now

* cmake installs helper to a wrong place and I
didn't manage to change it with KAUTH_HELPER_INSTALL_DIR

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a5e60e1..939641d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -365,9 +365,9 @@ add_executable(kauth_ktexteditor_helper buffer/katesecuretextbuffer.cpp)
 target_link_libraries(kauth_ktexteditor_helper
   KF5::Auth
 )
-install(TARGETS kauth_ktexteditor_helper DESTINATION ${KAUTH_HELPER_INSTALL_DIR} )
-kauth_install_helper_files(kauth_ktexteditor_helper org.kde.ktexteditor.katetextbuffer root)
-kauth_install_actions(org.kde.ktexteditor.katetextbuffer buffer/org.kde.ktexteditor.katetextbuffer.actions)
+#install(TARGETS kauth_ktexteditor_helper DESTINATION ${KAUTH_HELPER_INSTALL_DIR} )
+#kauth_install_helper_files(kauth_ktexteditor_helper org.kde.ktexteditor.katetextbuffer root)
+#kauth_install_actions(org.kde.ktexteditor.katetextbuffer buffer/org.kde.ktexteditor.katetextbuffer.actions)
 
 # add part
 add_subdirectory(part)
-- 
2.15.0

